roles/internal/Islandora-Devops.alpaca/tasks/compile.yml

```diff
@@ -1,45 +1,15 @@
 ---
 
-- name: Create maven repository directory
-  file:
-    state: directory
-    path: "{{ alpaca_local_mvn_path }}"
-
-- name: Create maven repository directory
-  file:
-    state: directory
-    path: "~/.m2"
-
-- name: Drop in templated Maven settings
-  template:
-    src: maven.xml.j2
-    dest: "~/.m2/settings.xml"
-
 - name: Clone CLAW Alpaca
   git:
-    repo: https://github.com/Islandora-CLAW/Alpaca.git
+    repo: https://github.com/whikloj/Alpaca.git
     dest: "{{ alpaca_clone_directory }}"
     version: "{{ alpaca_version }}"
   register: alpaca_git
 
 - name: Install Alpaca via Gradle
-  command: ./gradlew install
+  command: ./gradlew build shadowJar
   args:
     chdir: "{{ alpaca_clone_directory }}"
   when: alpaca_git.changed
```

Comment:
Should... this be run as a non-root user? Seems like it will create/leave the "gradle daemon" dealio intact running as root... seems less-than ideal?

Reply:
Changed to run as the `alpaca_user`, in https://github.com/Islandora-Devops/islandora-playbook/pull/208/commits/a7414625efea5868dd0d22598cee559e7928003c